Banking/src/main/java/com/example/Banking/exception/IdNotFoundException.java

```diff
@@ -0,0 +1,21 @@
+package com.example.Banking.exception;
+
+import lombok.Data;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Data
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+@Component
+public class IdNotFoundException extends Exception {
```

Comment:
Use lombok's AllArgConstructor and NoArgConstructor instead of creating constructors manually

Reply:
Sure. I will change.